src/components/helper-tool.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { CSSProperties } from "react";
+import { XMarkIcon } from '@heroicons/react/20/solid'
+
+type HelperToolProps = {
+  data: Record<string, any>;
+  locale?: string;
+};
+
+export const HelperTool = ({ data, locale }: HelperToolProps) => {
+ if (!data){
+         return  <div></div>
```

Comment:
I think you can just return null. Or else `<></>` if that doesn't work. No need to make an empty div.